src/components/Molecules/Dropdown/Dropdown.tsx

```diff
@@ -69,10 +70,15 @@ const DropdownWithoutState: FC<DropdownPropsEnhanced> = props => {
               {options[currentIndex].label}
             </StyledTypo>
           )}
-          <StyledIcon width={16} height={16} rotate={showDropdown ? 270 : 90}>
-            <SvgEdit />
-          </StyledIcon>
-
+          {icon ? (
+            <StyledIcon width={16} height={16} rotate={0}>
+              {icon}
+            </StyledIcon>
+          ) : (
+            <StyledIcon width={16} height={16} rotate={showDropdown ? 270 : 90}>
+              <SvgEdit />
+            </StyledIcon>
+          )}
```

Comment:
I'd suggest to just pass the actual icon in thats required instead of toggling between them.

```suggestion
            <StyledIcon width={16} height={16} rotate={0}>
              {svgIcon}
            </StyledIcon>
```